app/assets/javascripts/application.js

```diff
@@ -3,7 +3,7 @@
 //= require linkify
 //= require jquery.easing
 //= require jquery.blockUI
-//= require jquery.elastic.source
+//= require jquery.elastic
```

Comment:
Sometime asset pipeline apparently was updated so it doesn't like files that end in "source"? Changing this filename made it suddenly find it again in current versions of the gem.